demo/src/screens/componentScreens/PickerScreen.tsx

```diff
@@ -128,13 +128,12 @@ export default class PickerScreen extends Component {
           </Picker>
 
           <Picker
-            title="Native Picker"
+            label="Native Picker"
```

Comment:
This removed the text since it's not migrated

Reply:
Fixed :)